tests/cases/string/out

```diff
@@ -57,9 +57,8 @@
     [${mkSectionName sectName}]
   ''
   ###
-  ''-couch_ini ${cfg.package}/etc/default.ini ${configFile} ${
-    pkgs.writeText "couchdb-extra.ini" cfg.extraConfig
-  } ${cfg.configFile}''
+  ''-couch_ini ${cfg.package}/etc/default.ini ${configFile} ${pkgs.writeText "couchdb-extra.ini"
+    cfg.extraConfig} ${cfg.configFile}''
```

Comment:
when interpolating something big, like an attribute set or an if-then-else, this has the potential to create very long lines, which can be ugly as well

I would say that we should leave it as-is, with/without this change strings interpolations can be ugly in some scenarios of real life, but the current alternative at least try to keep things in a small horizontal viewport of --max-width

What are your thoughts on this?

Reply:
trying to see what it looks like incorporating drain_comments_and_newlines. Preserve the intention from author to split, otherwise inline. Not 100% sure yet

Reply:
I just incorporated this behaviour into main, so this PR is not necessary anymore